docs/source/pythonapi/examples/tally-arithmetic.ipynb

```diff
@@ -0,0 +1,1512 @@
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    nuclide    score      mean  std. dev.\n",
+       "bin                                      \n",
+       "0     total  fission  1.080124   0.008126"
+      ]
+     },
+     "execution_count": 28,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Compute fast fission factor factor using tally arithmetic\n",
+    "therm_fiss_rate = sp.get_tally(name='therm. fiss. rate')\n",
+    "tot_fiss_rate = sp.get_tally(name='tot. fiss. rate')\n",
+    "fast_fiss = tot_fiss_rate / therm_fiss_rate\n",
+    "fast_fiss.get_pandas_dataframe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The thermal flux utilization is calculated as\n",
+    "$$f=\\frac{\\Sigma_{a,thermal}^F\\phi}{\\Sigma_a\\phi}$$\n",
+    "where $F$ denotes fuel."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div style=\"max-height:1000px;max-width:1500px;overflow:auto;\">\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>nuclide</th>\n",
+       "      <th>score</th>\n",
+       "      <th>mean</th>\n",
+       "      <th>std. dev.</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>bin</th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>total</td>\n",
+       "      <td>absorption</td>\n",
+       "      <td>0.769263</td>\n",
+       "      <td>0.004172</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    nuclide       score      mean  std. dev.\n",
+       "bin                                         \n",
+       "0     total  absorption  0.769263   0.004172"
+      ]
+     },
+     "execution_count": 29,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Compute thermal flux utilization factor using tally arithmetic\n",
+    "fuel_therm_abs_rate = sp.get_tally(name='fuel therm. abs. rate')\n",
+    "therm_util = fuel_therm_abs_rate / abs_rate\n",
+    "therm_util.get_pandas_dataframe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's move on to a more complicated example now. Before we set up tallies to get reaction rates in the fuel and moderator in two energy groups for two different nuclides. We can use tally arithmetic to divide each of these reaction rates by the flux to get microscopic multi-group cross sections."
```

Comment:
I added the fourth factor, but unfortunately one cannot multiply a bunch of derived tallies together.